### PR TITLE
Added a new theme for Kanboard - ThemeMaestro

### PR DESCRIPTION
**Pull request by JustFxDev**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1869,6 +1869,23 @@
         "title": "Telegram",
         "version": "1.3.3"
     },
+    "ThemeMaestro": {
+        "author": "JustFxDev (Fx)",
+        "compatible_version": ">=1.2.11",
+        "description": "Nice and colored theme family for Kanboard. Supports the Customizer plugin.",
+        "download": "https://github.com/JustFxDev/ThemeMaestro/blob/main/release/v0.7.0/ThemeMaestro.zip",
```
Comment (aljawaid) [inline plugins.json]: I'm not sure if this link will install or parse correctly as opening it should trigger a download, but instead it goes to the GitHub page.

I think the file structure is wrong for this plugin. Check the structure in the readme of this repository.

**Review by aljawaid — CHANGES_REQUESTED** [review]
Look into the file structure and the download link

**Review by aljawaid — APPROVED** [review]
This link looks fine now, but the plugin file structure still seems wrong, but if it works it works.

File: plugins.json [hunk 2]
```diff
@@ -1890,7 +1890,7 @@
         "author": "greyaz",
         "compatible_version": ">=1.2.8",
         "description": "A 'task-first' and high-quality theme for Kanboard. It's aimed at better mobile experiences, common plugins' compatibilities, and customization friendly.",
-        "download": "https://github.com/greyaz/ThemeRevision/releases/download/1.1.9/ThemeRevision.zip",
+        "download": "https://github.com/JustFxDev/ThemeMaestro/raw/main/release/v0.7.0/ThemeMaestro.zip",
```
Comment (aljawaid) [inline plugins.json]: I would have used this link, but if yours works, it works.

`https://github.com/JustFxDev/ThemeMaestro/releases/download/v0.7.0/ThemeMaestro_v0.7.0.zip`

**Comment by JustFxDev** [comment]
> This link looks fine now, but the plugin file structure still seems wrong, but if it works it works.

There is no file structure. Just the main folder and some files in that one. It's a theme. 

For sure i can put the css in subfolders but i think this is not neccessary.

Is there something else to do? I saw it hast not been pulled.

**Comment by aljawaid** [comment]
I think the subfolders matter for updating all versions.

**Comment by JustFxDev** [comment]
> I think the subfolders matter for updating all versions.

Sorry, i do not understand what this means. If the archive is unpacked to .\plugins one gets

```
.\plugins\ThemeMaestro\ThemeMaestro.css
.\plugins\ThemeMaestro\blue.css
.\plugins\ThemeMaestro\column_colors_multi.css
.\plugins\ThemeMaestro\column_colors_none.css
.\plugins\ThemeMaestro\green.css
.\plugins\ThemeMaestro\LICENSE
.\plugins\ThemeMaestro\Plugin.php
.\plugins\ThemeMaestro\README.md
.\plugins\ThemeMaestro\red.css
 ```
 containing all the files.
 
 Same as e. g. in the WekanBoard Theme.
 
 Please let me know what is expected here. I simply do not get it.

**Comment by aljawaid** [comment]
> > I think the subfolders matter for updating all versions.
> 
> Sorry, i do not understand what this means. If the archive is unpacked to .\plugins one gets
> 
> ```
> .\plugins\ThemeMaestro\ThemeMaestro.css
> .\plugins\ThemeMaestro\blue.css
> .\plugins\ThemeMaestro\column_colors_multi.css
> .\plugins\ThemeMaestro\column_colors_none.css
> .\plugins\ThemeMaestro\green.css
> .\plugins\ThemeMaestro\LICENSE
> .\plugins\ThemeMaestro\Plugin.php
> .\plugins\ThemeMaestro\README.md
> .\plugins\ThemeMaestro\red.css
> ```
> 
> containing all the files.
> 
> Same as e. g. in the WekanBoard Theme.
> 
> Please let me know what is expected here. I simply do not get it.

add me to your repository or make your folder structure like this:
https://github.com/kanboard/website#folder-structure

Also, I think you need to update your branch/rebase

**Comment by JustFxDev** [comment]
> add me to your repository

done

**Comment by aljawaid** [comment]
close this pull request, let me clean up the structure and then we can do another pull request

**Pull request closed without merging**